package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "Framework to automate development of Algorand Assets and Smart Contracts.",
   "repository": "https://github.com/scale-it/algorand-builder.git",
-  "author": "Robert Zaremba <https://github.com/robert-zaremba>",
+  "author": "Robert Zaremba <https://github.com/robert-zaremba>, Martynas Maciulevičius <https://github.com/Invertisment>",
```

Comment:
I think we need to move this things to `contirbutors` https://flaviocopes.com/package-json/#contributors